utils/stm.stanza

```diff
@@ -6,58 +7,72 @@ defpackage ocdb/stm:
   import reader
   import jitx
 
+defstruct StmError <: Exception : 
+  message:?
```

Comment:
message could be anything, right? Do not we want to be more restrictive?

Reply:
This allows adding more detail to the type incrementally, right now it's just String|Printable

Reply:
Fixed, and made public